modules/picotool/2.1.0/MODULE.bazel

```diff
@@ -0,0 +1,26 @@
+module(
+    name = "picotool",
+    version = "2.1.0",
+)
+
+bazel_dep(name = "rules_libusb", version = "0.1.0-rc1")
+bazel_dep(name = "pico-sdk", version = "2.1.0")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "bazel_skylib", version = "1.6.1")
+bazel_dep(name = "rules_python", version = "0.22.1")
+bazel_dep(name = "platforms", version = "0.0.9")
+
+libusb = use_extension("@rules_libusb//:extensions.bzl", "libusb")
+libusb.source_release(min_version = "1.0.22")
+use_repo(libusb, "libusb")
+
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# TODO: Upstream a bazel build
```

Comment:
FYI mbedtls is already in the BCR, although I ran into some trouble trying to use it with Pigweed (https://pwbug.dev/354749299.